client/src/index.css

```diff
@@ -1,18 +1,24 @@
 * {
-  font-family: 'Airbnb Cereal App', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
-    Ubuntu, 'Helvetica Neue', sans-serif;
+  margin: 0;
+  padding: 0; 
+  box-sizing: border-box;
 }
 
-:root {
-  --primary: #FF385C;
-  --black: #222222;
-  --grey: #717171;
-  --gradient: linear-gradient(to right, #E61E4D 0%, #E31C5F 50%, #D70466 100%);
+html {
+  font-size: 10px;
```

Comment:
The only issue I see here is if you use anything that modifies the root font size, it will override the HTML selector as it has a higher (the highest) specificity.  As long as you keep this in mind and make note of whether it operates as expected, this should be fine.

Reply:
Care to show me an example where the root font size would be changed? Im not following at the moment.